config/hooks/normal/1000-create-mtab-symlink.hook.chroot

```diff
@@ -0,0 +1 @@
+/usr/share/live/build/hooks/normal/1000-create-mtab-symlink.hook.chroot
```

Comment:
Why are all these symlinks needed?  Are they not added by lb config?

Reply:
That is the original code, Maybe I bumped it by accident.
Our scripts are in the 0500-0599 range.